### PR TITLE
Vertex project guidelines

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description

Add vertex-project-guidelines.md

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+2. You can configure the Vertex app so that the invoice is saved in Vertex. However, we recommend to send invoice requests only for paid orders, as specified in [Vertex installation](https://docs.spryker.com/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/install-vertex.html#optional-if-you-plan-to-send-invoices-to-vertex-through-oms-configure-your-payment-oms). The current implementation works asynchronously, so no response is saved in Spryker.
+
+3. When using Vertex for tax determination, no exact tax rate is received from Vertex instead of a default tax value in the Back Office. So, to avoid confusion, we recommend removing the default tax rate that appears in the Back Office.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md:35]: What do you mean by the default tax rate? Do you mean the tax rates that are shipped with the demo shops?

File: docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Failover Solution
+The integration covers use cases for when there is a downtime and for reasons taxes cannot be calculated. In this scenario, no tax is displayed and the end-user can checkout without taxes.
+
+In the future, we will include a flag that states: VERTEXDOWN – 0 rate to enable you differentiate the downtime scenario from when tax rates are actually calculated as 0 from Vertex.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md:40]: How critical is it to provide this information? Docs don't like future promises very much

File: docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+2. You can configure the Vertex app so that the invoice is saved in Vertex. However, we recommend to send invoice requests only for paid orders, as specified in [Vertex installation](https://docs.spryker.com/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/install-vertex.html#optional-if-you-plan-to-send-invoices-to-vertex-through-oms-configure-your-payment-oms). The current implementation works asynchronously, so no response is saved in Spryker.
+
+3. When using Vertex for tax determination, no exact tax rate is received from Vertex instead of a default tax value in the Back Office. So, to avoid confusion, we recommend removing the default tax rate that appears in the Back Office.
+
+## Failover Solution
+The integration covers use cases for when there is a downtime and for reasons taxes cannot be calculated. In this scenario, no tax is displayed and the end-user can checkout without taxes.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md:38]: The first sentence is a bit unclear. Break this down or rewrite for more clarity

File: docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md (hunk 1 above)
Comment (NifemiOmotola) [reply docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md:35]: yes this is correct.

File: docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md (hunk 3 above)
Comment (NifemiOmotola) [reply docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md:38]: Can you help here? I am referring to scenarios where the Vertex App is down. What would you suggest for a rewrite?

File: docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md (hunk 2 above)
Comment (NifemiOmotola) [reply docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex-project-guidelines.md:40]: I think it is important to let them know that tax rates can be 0 for different reasons. If you can help reword. Also fine

**Pull request merged**